### ISSUE TITLE
Explore methods to add Grants to Individuals to the standard

### ISSUE THREAD

**Issue by mrshll1001** [ThreeSixtyGiving/standard#332]
Following the Grants to Individuals proposal, this is to track progress with experimenting on implementation techniques to add Grants to Individuals to the standard.

**Comment by michaelwood** [comment]
In the grants to individuals conditionality 332 the type isn't specified which I think is causing flatten tool to error out

```
Fetching https://docs.google.com/spreadsheets/d/12fXtDOIbkYM7uHDh43x2td_gbLbpDSbKhTB98sUeyY0/export?format=xlsx
Running convert on data/original/a001p00000zgyHZAAY.xlsx to data/json_all/a001p00000zgyHZAAY.json


Unflattening failed for file data/original/a001p00000zgyHZAAY.xlsx

Traceback (most recent call last):
  File "/home/michael/dev/ods/threesixtygiving/datagetter/getter/get.py", line 205, in fetch_and_convert
    convert_spreadsheet(
  File "/home/michael/dev/ods/threesixtygiving/datagetter/getter/get.py", line 67, in convert_spreadsheet
    flattentool.unflatten(
  File "/home/michael/dev/ods/threesixtygiving/datagetter/.ve/lib/python3.8/site-packages/flattentool/__init__.py", line 317, in unflatten
    ) = spreadsheet_input.fancy_unflatten(
  File "/home/michael/dev/ods/threesixtygiving/datagetter/.ve/lib/python3.8/site-packages/flattentool/input.py", line 456, in fancy_unflatten
    cell_tree = self.do_unflatten()
  File "/home/michael/dev/ods/threesixtygiving/datagetter/.ve/lib/python3.8/site-packages/flattentool/input.py", line 407, in do_unflatten
    unflattened = unflatten_main_with_parser(
  File "/home/michael/dev/ods/threesixtygiving/datagetter/.ve/lib/python3.8/site-packages/flattentool/input.py", line 1016, in unflatten_main_with_parser
    raise ValueError(
ValueError: There is an object or list at 'recipientIndividual' but it should be an string
```


https://github.com/ThreeSixtyGiving/standard/blob/332-grants-to-individuals-conditionality/schema/360-giving-schema.json#L1218


This is seen on http://dqt-332-grants-to-individuals-conditionality.threesixtydokku2.vs.mythic-beasts.com/results/0c6160e6-6d52-4e9d-95c8-e772e51bc7eb# 

I think this needs a data type in order to flatten data successfully

**Comment by mrshll1001** [comment]
Thanks Michael, I've added a data type of object now.

**Comment by mariongalley** [comment]
I think this can be closed now as it's fixed by #339 @mrshll1001
